src/jdk.jdwp.agent/share/native/libjdwp/ModuleReferenceImpl.c

```diff
@@ -50,6 +50,11 @@ getName(PacketInputStream *in, PacketOutputStream *out)
         return JNI_TRUE;
     }
     namestr = (jstring)JNI_FUNC_PTR(env, CallObjectMethod) (env, module, method);
+    if (JNI_FUNC_PTR(env,ExceptionCheck)(env)) {
+        JNI_FUNC_PTR(env,ExceptionClear)(env); // keep -Xcheck:jni happy
+        ERROR_MESSAGE(("JNI Exception occurred calling Module.getName()"));
+        EXIT_ERROR(AGENT_ERROR_JNI_EXCEPTION, NULL);
```

Comment:
Don't you want to report what the exception was?

Reply:
That might be nice, but we don't do it in other places where we have this pattern. This call should never fail, thus the "fatal error" approach.

Reply:
Sure but when it does fail because of a bug the person trying to debug it will lament the fact they don't know what exception was encountered. Seems a little helper function could do the above and call `ExceptionDescribe`. Future RFE I guess.

Reply:
I was thinking the same. I'll file an RFE.

Reply:
Filed [JDK-8345515](https://bugs.openjdk.org/browse/JDK-8345515)